src/randoop/main/GenInputsAbstract.java

```diff
@@ -219,20 +184,18 @@ public GenInputsAbstract(String command, String pitch,
   @OptionGroup("Values used in tests")
   @Option("Use null as an input with the given frequency")
   public static double null_ratio = 0;
-  
+
   /**
-   * Never use null as input to methods or constructors.
+   * Suppress use of null when no other argument value can be generated.
    * 
-   * This option causes Randoop to abandon the method call rather than providing
-   * null as an input, when no non-null value of the appropriate type is available.
+   * If true, Randoop will fail input generation rather than use null as an 
```

Comment:
What does "fail input generation" mean?  A user might imagine that Randoop crashes or has some other behavior.  Please make the text explicit so that users understand what is happening and the consequences of their choices, in words that are linked to concepts in the Randoop manual and that they can understand.